src/main/java/net/jeqo/gizmo/Managers/Commands/GizmoCommandManager.java

```diff
@@ -0,0 +1,72 @@
+package net.jeqo.gizmo.Managers.Commands;
```

Comment:
I'm not too sure how I feel about this command manager, as I am doing a repository restructure I will keep this change in mind, but I already have a sweet command manager in mind.